test/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# setup tiny-cnn
```

Comment:
this file does not really describe a test, more an example file no ? Not sure what the file does but maybe you want to use CMake's add_test.